### ISSUE TITLE
How does the "cli" object work?

### ISSUE THREAD

**Issue by pa9ey** [yegor256/eo#7]
With regard to the Hello, World example:

```
import org.eolang.printed;
import org.eolang.string;
cli(
  printed(
    string("Hello, world!")
  )
)
```

I was sort of expecting to see a method call on the `cli` object and wondered whether, given that there isn't one, maybe some "work" happens in the constructor.

Yegor's answer was "The idea is that cli object is highest-level one in the script and it starts automatically."

Transferred from the comments on [this commit](https://github.com/yegor256/eo/commit/51b13926420ecc9cb30b9ab43a9846af782513fb).

**Comment by DronMDF** [comment]
This look like a functions...
printed and string too...

```
create app as Application {
    Application()
        @(create printer as Printer, create hello("Hello world") as String)
    execute() {
           printer.print(hello);
     }
}
```

Who execute app? We cannot write code without object.

May be need to intoduce precreated system object, that can execute app?

**Comment by pa9ey** [comment]
I think an outcome for this issue is likely to be addressed by #58 so if there are no objections I will close this issue.
